bpf/lib/policy.h

```diff
@@ -36,7 +46,7 @@
 #ifdef REQUIRES_CAN_ACCESS
 static inline bool identity_is_reserved(__u32 identity)
 {
-	return identity < MINIMAL_NUMERIC_IDENTITY;
+	return (identity & NUMERIC_IDENTITY_MASK) == 0;
```

Comment:
Won't this perform 2 operations instead of 1 like before?

Reply:
Yep. I think at some point I must have been looking at an option which would have created two if conditions, which I tried to avoid. However that's not in this patch so there's no reason for this to be so complicated.